src/util/fetchData.js

```diff
@@ -0,0 +1,17 @@
+import axios from "axios";
+
+export const login = async ({ email, password }) => {
```

Comment:
Stretch Goal: There will be a lot of repetition here. How can we leverage classes to help clean things up while making everything accessible?